src/main/java/com/bina/varsim/tools/evaluation/VCFcompare.java

```diff
@@ -1507,6 +1507,10 @@ public int compareVariant(Variant variant, int genotype, BitSet validated) {
                 }
             } else {
                 // Non-SNPs
+                if ((type ==  VariantType.Insertion || type == VariantType.Deletion || type == VariantType.Complex ) &&
+                    intervalForCompare.right - intervalForCompare.left < Constant.SVLEN) {
```

Comment:
could you do `+1` for 1-based interval?

Reply:
also could you update the help for `-wig` option, so that future users are aware of this change? Thanks!

Reply:
Ok. Sounds good. Thank you for the reminder!